### PR TITLE
osd: remove redundant options to enable bluestore

### PR DESCRIPTION
**Pull request by leseb**
There is no need for 2 variables to enable bluestore, prior to this
patch one had to do the following to activate bluestore:

osd_objectstore: bluestore
bluestore: true

Now you just need to set `osd_objectstore: bluestore`.

Fixes: https://github.com/ceph/ceph-ansible/issues/1475, https://github.com/ceph/ceph-ansible/issues/1393 and https://github.com/ceph/ceph-ansible/issues/1423
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test kraken-ansible2.2-journal_collocation

**Comment by leseb** [comment]
tested and validated locally, we don't have any test for this now. Working on it.

**Comment by leseb** [comment]
@fullerdj

**Comment by leseb** [comment]
@andrewschoen can you help me adding the test in the CI please?
Thanks!

File: roles/ceph-osd/defaults/main.yml [hunk 1]
```diff
@@ -122,12 +132,48 @@ raw_multi_journal: false
...
 raw_journal_devices: []
 
 
 # IV. This will partition disks for BlueStore
-# Use 'true' to enable this scenario
-bluestore: false
+# To enable bluestore just set:
+# osd_objectstore: bluestore
+#
+# If osd_objectstore: bluestore is enabled, by default if empty bluestore_wal_devices
```
Comment (guits) [inline roles/ceph-osd/defaults/main.yml]: this phrase is weird

**Comment by guits** [comment]
jenkins test jewel-ansible2.2-dmcrypt_journal

**Comment by guits** [comment]
@leseb @andrewschoen shouldn't we rename these directories:

from : `tests/functional/centos/7/journal-collocation-bluestore`
to : `tests/functional/centos/7/bluestore-journal-collocation`

from : `tests/functional/centos/7/cluster-bluestore`
to : `tests/functional/centos/7/bluestore-cluster` 

to keep consistency regarding the existing directories and scenarios names ?
eg. :
```
tests/functional/centos/7/docker-cluster-dmcrypt-journal-collocation
tests/functional/centos/7/docker-cluster
...
ansible2.2-centos7_cluster
ansible2.2-xenial_cluster
ansible2.2-docker_dmcrypt_journal_collocation
...
```

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-purge_cluster

**Comment by leseb** [comment]
jenkins test kraken-ansible2.2-centos7_cluster

**Comment by leseb** [comment]
jenkins test kraken-ansible2.2-update_cluster

**Comment by leseb** [comment]
jenkins test kraken-ansible2.2-xenial_cluster

**Comment by leseb** [comment]
jenkins test this please

**Pull request merged**